packages/odyssey-design-tokens/src/color/palette.json

```diff
@@ -8,105 +8,125 @@
-          "value": "#f5f5f6"
+          "value": "#F4F4F4"
         },
         "100": {
           "comment": "Used for read-only/disabled states",
-          "value": "#ebebed"
+          "value": "#EDEDED"
+        },
+        "200": {
+          "comment": "Used for Display borders",
+          "value": "#E1E1E1"
         },
-        "200": { "comment": "Used for Display borders", "value": "#d7d7dc" },
         "500": {
           "comment": "Used for Interactive UI borders",
-          "value": "#8c8c96"
+          "value": "#8D8D8D"
         },
         "600": {
           "comment": "Used for caption text, Status label, disabled Overlay Buttons",
-          "value": "#6e6e78"
+          "value": "#6E6E6E"
         },
         "900": {
           "comment": "Used for titles, body copy, icons, shadows, tooltips",
-          "value": "#1d1d21"
+          "value": "#272727"
         }
       },
       "blue": {
         "000": {
           "comment": "Used for clear button hover, backgrounds",
-          "value": "#f2f5ff"
+          "value": "#F2F3FD"
         },
         "300": {
           "comment": "Used for buttons disabled state fills & borders, focus rings",
-          "value": "#a7b5ec"
-        },
-        "400": {
-          "comment": "Used for a11y-friendly UI backgrounds",
-          "value": "#788ee2"
+          "value": "#9DAAF1"
         },
         "500": {
           "comment": "Used for links, buttons, icons, hover states, selected states",
-          "value": "#1662dd"
+          "value": "#4C64E1"
         },
         "900": {
           "comment": "Used for okta logo, brand moments, button hovers",
-          "value": "#00297a"
+          "value": "#22307C"
         }
       },
       "green": {
         "000": {
           "comment": "Used for moments of success, validation, wayfinding",
-          "value": "#e5faf3"
+          "value": "#DEFAE7"
+        },
+        "300": {
+          "comment": "N/A",
+          "value": "#7BE09E"
         },
-        "300": { "comment": "N/A", "value": "#84d2b1" },
         "500": {
           "comment": "Used for moments of success, validation, wayfinding, status",
-          "value": "#00b478"
+          "value": "#197F48"
         },
         "600": {
-          "comment": "Used for background color for success messaging",
-          "value": "#008055"
+          "comment": "N/A",
+          "value": "#0E562F"
         },
         "900": {
           "comment": "Used for moments of success, validation, wayfinding",
-          "value": "#00503c"
+          "value": "#0A4023"
         }
       },
       "red": {
         "000": {
           "comment": "Used for system errors, danger buttons, error states",
-          "value": "#ffdfe2"
+          "value": "#FFF0EE"
         },
         "300": {
           "comment": "Used for danger button disabled state fills, borders, & focus ring",
-          "value": "#f88c90"
+          "value": "#FEBBAE"
```

Comment:
Are the capital letters easier to copy outta Figma? :P

Reply:
I'm going to downcase all of these before merge. 

Right now, it's useful for me to see which values are deviations and match them against Figma's output.